### PR TITLE
00044 update readme

### PR DESCRIPTION
**Pull request by SimiHunjan**
**Description**:

Update readme

**Related issue(s)**:

Fixes #
https://github.com/hashgraph/hedera-local-node/issues/44

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,112 @@
 [![npm (tag)](https://img.shields.io/npm/v/@hashgraph/hedera-local)](https://www.npmjs.com/package/@hashgraph/hedera-local)
 
-# @hashgraph/hedera-local
-
-Developer tooling for running a Local Hedera Network (Consensus + Mirror Nodes)
+The Hedera Local Node project allows developers to set up their own local network. The local network is composed of one mirror node and one consensus node. You can set this up by either using the CLI tool or by running Docker.
 
-## What
+Note: This currently does not map to the latest mirror node version or mainnet consensus node version. An issue has been opened to address this.
```
Comment (Neeharika-Sompalli) [inline README.md:5]: May be link the issue as well ?

File: README.md [hunk 2]
```diff
@@ -1,34 +1,112 @@
...
+### Note:
 
 - Ensure to use Docker Compose version 1.29.2 on macOS, due to known bug in Docker Compose V2.
 - Ensure the `gRPC FUSE for file sharing` and `Use Docker Compose V2` settings are disabled in the docker settings.
 
 ![docker-compose-settings.png](docker-compose-settings.png)
 
+- Ensure the following configurations are set in Docker **Settings** -> **Resources**
+    - **CPUs:** 6
+    - **Memory:** 5GB
```
Comment (Neeharika-Sompalli) [inline README.md:32]: May be mention `atleast` these resource usage to be available for docker compose.

File: README.md [hunk 3]
```diff
@@ -54,6 +132,8 @@ Available commands:
 
 ### Commands
 
+Note: There are known issues with this command
```
[the remark this answers is not in the document]
Comment (SimiHunjan) [reply README.md]: Thanks for the feedback. Linked the bug issues for this command.

File: README.md [hunk 4]
```diff
@@ -1,34 +1,112 @@
...
 ## Requirements
 
 - [Node.js](https://nodejs.org/) >= v16.x and npm >= v8.5.x
+     - Node version check: `node -v`
+     - NPM version check: `npm -v`
+
 - [Docker](https://www.docker.com/) >= v20.10.x
+    - Docker version check: `docker -v`
 - [Docker Compose](https://docs.docker.com/compose/) >= v1.25.x
+     - Docker Compose version check: `docker-compose -v`
```
[the remark this answers is not in the document]
Comment (SimiHunjan) [reply README.md:21]: Thanks for the feedback. Added the bug issues for this command.

File: README.md (hunk 1 above)
Comment (SimiHunjan) [reply README.md]: Thanks linked!

File: README.md (hunk 2 above)
Comment (SimiHunjan) [reply README.md:32]: thanks added

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (JonathanGiles) [reply README.md:21]: I'm not seeing this being resolved yet?

**Review by Neeharika-Sompalli — COMMENTED** [review]
@SimiHunjan Just a question, Will you also be adding an example how to use with java sdk ?

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM ! Other than question regarding an example on using with java sdk

**Comment by SimiHunjan** [comment]
@Neeharika-Sompalli I was going to publish the instructions for use with the SDKs in the docs site and then link it here in the readme. I was thinking I would maintain those SDK examples there instead of the readme.

**Pull request merged**